mlir/lib/Conversion/TosaToLinalg/TosaToLinalgPass.cpp

```diff
@@ -90,7 +90,6 @@ void mlir::tosa::addTosaToLinalgPasses(
   pm.addNestedPass<func::FuncOp>(tosa::createTosaLayerwiseConstantFoldPass(
       {options.aggressiveReduceConstant}));
   pm.addNestedPass<func::FuncOp>(tosa::createTosaMakeBroadcastablePass());
-  pm.addNestedPass<mlir::ModuleOp>(
-      tosa::createTosaValidation(validationOptions));
+  pm.addNestedPass<func::FuncOp>(tosa::createTosaValidation(validationOptions));
```

Comment:
Why not simply `pm.addPass(tosa::createTosaValidation(validationOptions));` ?

Reply:
done